distributions/openhab-demo/src/main/resources/userdata/config/org/openhab/addons.config

```diff
@@ -0,0 +1,8 @@
+:org.apache.felix.configadmin.revision:=L"1"

```

Comment:
I'm wondering if it is advisable to include this file in a demo package. People might get the impression that this file is meant to be manually created/maintained.
Wouldn't it be better to stick to the `addons.cfg` file? Yes, the downside would be that people cannot easily change the installed add-ons, but if we consider the demo package to also be an example of how to package openHAB configuration as text files, it would imho more appropriate.

Reply:
OK no problem for me. I will just put an overlay on top of this overlay to make it workable for me. :smile: